nx-X11/lib/X11/XlibInt.c

```diff
@@ -2751,6 +2751,25 @@ void _XEatData(
 #undef SCRATCHSIZE
 }
 
+/*
+ * Read and discard "n" 32-bit words of data
+ * Matches the units of the length field in X protocol replies, and provides
+ * a single implementation of overflow checking to avoid having to replicate
+ * those checks in every caller.
+ */
+#ifndef HAVE__XEATDATAWORDS
```

Comment:
Actually, we stumbled over a libXrender originated crash in Ubuntu 14.04 which I cannot reproduce on Debian jessie (not got to testing on Debian unstable). In that crash, we end up in libX11's _Xsend() function, because libXrender calls a function not shipped in libNX_X11. The code path then switches over to libX11 at some point (I think it was the _XGetRequest function we things went astray) and never returns into libNX_X11's objects.